### ISSUE TITLE
🐏 ramda typing woes

### ISSUE THREAD

**Issue by astroash** [TwinePlatform/twine-monolith#211]
<img width="869" alt="Screen Shot 2019-07-02 at 15 35 08" src="https://user-images.githubusercontent.com/14337958/60509863-03fc0980-9cdf-11e9-99d1-efd88c8cb1b5.png">

**Comment by eliascodes** [comment]
This is fixed in #209 and #210

**Comment by eliascodes** [comment]
Happening again on #227

**Comment by astroash** [comment]
I genuinely think that yarn workspaces will solve this issue as it hoists packages to the root level

**Comment by eliascodes** [comment]
Yeah this has been annoying enough now, I'm happy to try moving to yarn.
